apps/token-e2e/src/integration/view/staking.cy.js

```diff
@@ -56,37 +66,70 @@ context('Staking Page - verify elements on page', function () {
       });
 
       it('Should be able to see validator stake', function () {
-        cy.get('[col-id="stake"] > div > span')
+        cy.get('[col-id="stake"] > div > span > span')
           .should('have.length.at.least', 1)
           .each(($stake) => {
             cy.wrap($stake).should('not.be.empty');
           });
       });
 
-      it('Should be able to see validator normalised voting power', function () {
-        cy.get('[col-id="normalisedVotingPower"] > div > span')
-          .should('have.length.at.least', 1)
-          .each(($vPower) => {
-            cy.wrap($vPower).should('not.be.empty');
-          });
+      it('Should be able to see validator stake tooltip', function () {
+        cy.get('[col-id="stake"] > div > span > span').first().realHover();
+
+        cy.get(stakedByOperatorToolTip)
+          .invoke('text')
+          .should('contain', 'Staked by operator: 0.00');
+        cy.get(stakedByDelegatesToolTip)
+          .invoke('text')
+          .should('contain', 'Staked by delegates: 0.00');
+        cy.get(totalStakedToolTip)
+          .invoke('text')
+          .should('contain', 'Total stake: 0.00');
       });
 
       it('Should be able to see validator normalised voting power', function () {
-        cy.get('[col-id="normalisedVotingPower"] > div > span')
+        cy.get('[col-id="normalisedVotingPower"] > div > span > span')
           .should('have.length.at.least', 1)
           .each(($vPower) => {
             cy.wrap($vPower).should('not.be.empty');
           });
       });
 
+      it('Should be able to see validator voting power tooltip', function () {
+        cy.get('[col-id="normalisedVotingPower"] > div > span > span')
+          .first()
+          .realHover();
+
+        cy.get(unnormalisedVotingPowerToolTip)
+          .invoke('text')
+          .should('contain', 'Unnormalised voting power: 0.00%');
+        cy.get(normalisedVotingPowerToolTip)
+          .invoke('text')
+          .should('contain', 'Normalised voting power: 0.10%');
+      });
+
       it('Should be able to see validator total penalties', function () {
-        cy.get('[col-id="totalPenalties"] > div > span')
+        cy.get('[col-id="totalPenalties"] > div > span > span')
           .should('have.length.at.least', 1)
           .each(($penalties) => {
             cy.wrap($penalties).should('contain.text', '0%');
           });
       });
 
+      it('Should be able to see validator penalties tooltip', function () {
+        cy.get('[col-id="totalPenalties"] > div > span > span').realHover();
+
+        cy.get(performancePenaltyToolTip)
+          .invoke('text')
+          .should('contain', 'Performance penalty: 100.00%');
+        cy.get(overstakedPenaltyToolTip)
+          .invoke('text')
+          .should('contain', 'Overstaked penalty: NaN%');
```

Comment:
Chore (blocking): This is a NaN bug, so could you pull this test out for now please

Reply:
I've removed the assertion for NaN. 
Bug ticket here: https://github.com/vegaprotocol/frontend-monorepo/issues/2886